  INFO root [2019-09-15 09:35:31,716] - 
  INFO root [2019-09-15 09:35:31,717] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-15 09:35:31,718] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-15 09:35:31,718] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-15 09:35:31,719] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-15 09:35:31,735] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56376 )
  INFO root [2019-09-15 09:35:31,735] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56376, SOMAXCONN=5.
  INFO root [2019-09-15 09:35:31,736] - server-configs:
  INFO root [2019-09-15 09:35:31,736] - 	internalTcpAddr : 192.168.56.1:56376
  INFO root [2019-09-15 09:35:31,736] - 	componentID : 6000
  WARN root [2019-09-15 09:35:31,736] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-15 09:35:32,076] - Components::findLogger: found logger, addr:192.168.56.1:56374
  INFO root [2019-09-15 09:35:32,076] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 09:35:32,077] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 09:35:32,093] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 6480 ----
  INFO root [2019-09-15 09:35:32,100] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-15 09:35:32,103] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-15 09:35:32,103] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-15 09:35:32,532] - ---- cellappmgr is running ----
 DEBUG root [2019-09-15 09:35:32,534] - Components::process(): Request for the process of identity...
  WARN root [2019-09-15 09:35:38,654] - Forwarding logs to logger[192.168.56.1:56374/0/0/0]...
 ERROR root [2019-09-15 09:37:03,072] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=179726957534, lastReceivedTime=179723678290, inactivityExceptionPeriod=134887197)
  INFO root [2019-09-15 09:37:03,072] - The following logs sent to logger failed:
  WARN root [2019-09-15 09:37:03,072] - ==> [2019-09-15 09:37:03 070] TCPPacketReceiver::processPendingEvents(192.168.56.1:56374): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 09:37:03,072] - ==> [2019-09-15 09:37:03 073] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=179726957534, lastReceivedTime=179723678290, inactivityExceptionPeriod=134887197)
  WARN root [2019-09-15 09:37:03,072] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56375): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 09:37:03,072] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=179727197002, lastReceivedTime=179723783402, inactivityExceptionPeriod=134887197)
  INFO root [2019-09-15 09:37:03,072] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-15 09:37:03,072] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-15 09:37:03,072] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-15 09:37:03,072] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-15 09:39:20,254] - 
  INFO root [2019-09-15 09:39:20,258] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-15 09:39:20,259] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-15 09:39:20,260] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-15 09:39:20,260] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-15 09:39:20,283] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56417 )
  INFO root [2019-09-15 09:39:20,283] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56417, SOMAXCONN=5.
  INFO root [2019-09-15 09:39:20,283] - server-configs:
  INFO root [2019-09-15 09:39:20,283] - 	internalTcpAddr : 192.168.56.1:56417
  INFO root [2019-09-15 09:39:20,284] - 	componentID : 6000
  WARN root [2019-09-15 09:39:20,290] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-15 09:39:21,794] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 3148 ----
  INFO root [2019-09-15 09:39:21,795] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-15 09:39:21,796] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-15 09:39:21,796] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-15 09:39:22,191] - ---- cellappmgr is running ----
 DEBUG root [2019-09-15 09:39:22,195] - Components::process(): Request for the process of identity...
  INFO root [2019-09-15 09:39:29,337] - Components::findComponents: find logger(1)...
  INFO root [2019-09-15 09:39:29,373] - Components::findComponents: found logger, addr:192.168.56.1:56414
  INFO root [2019-09-15 09:39:29,373] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 09:39:29,374] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 09:39:29,375] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-15 09:39:29,378] - Components::findComponents: found dbmgr, addr:192.168.56.1:56415
  INFO root [2019-09-15 09:39:29,379] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 09:39:29,379] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 09:39:29,481] - Components::findComponents: found dbmgr, addr:192.168.56.1:56415
  WARN root [2019-09-15 09:39:30,482] - Forwarding logs to logger[192.168.56.1:56414/0/0/0]...
 ERROR root [2019-09-15 09:41:20,655] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=180306518147, lastReceivedTime=180305471027, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-15 09:41:20,660] - The following logs sent to logger failed:
  WARN root [2019-09-15 09:41:20,660] - ==> [2019-09-15 09:41:20 648] TCPPacketReceiver::processPendingEvents(192.168.56.1:56414): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 09:41:20,660] - ==> [2019-09-15 09:41:20 656] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=180306518147, lastReceivedTime=180305471027, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-15 09:41:20,767] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56415): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 09:41:20,770] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=180306777976, lastReceivedTime=180305536822, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-15 09:41:20,777] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-15 09:41:20,779] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-15 09:41:20,779] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-15 09:41:20,779] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-15 09:42:58,879] - 
  INFO root [2019-09-15 09:42:58,879] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-15 09:42:58,880] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-15 09:42:58,882] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/res/;D:/kbengine/BattleOfBalls_assets/scripts/
  INFO root [2019-09-15 09:42:58,882] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-15 09:42:58,910] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56459 )
  INFO root [2019-09-15 09:42:58,911] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56459, SOMAXCONN=5.
  INFO root [2019-09-15 09:42:58,911] - server-configs:
  INFO root [2019-09-15 09:42:58,911] - 	internalTcpAddr : 192.168.56.1:56459
  INFO root [2019-09-15 09:42:58,911] - 	componentID : 5129652375332859700
  WARN root [2019-09-15 09:42:58,911] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  WARN root [2019-09-15 09:42:59,252] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20603, errorno=10048
  INFO root [2019-09-15 09:42:59,256] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 5129652375332859700. UID: 2082. PID: 12364 ----
  INFO root [2019-09-15 09:42:59,256] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-15 09:42:59,256] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-15 09:42:59,257] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-15 09:42:59,647] - ---- cellappmgr is running ----
 DEBUG root [2019-09-15 09:42:59,647] - Components::process(): Request for the process of identity...
  INFO root [2019-09-15 09:43:02,408] - 
  INFO root [2019-09-15 09:43:02,423] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-15 09:43:02,424] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-15 09:43:02,424] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-15 09:43:02,424] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-15 09:43:02,452] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56465 )
  INFO root [2019-09-15 09:43:02,453] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56465, SOMAXCONN=5.
  INFO root [2019-09-15 09:43:02,453] - server-configs:
  INFO root [2019-09-15 09:43:02,453] - 	internalTcpAddr : 192.168.56.1:56465
  INFO root [2019-09-15 09:43:02,453] - 	componentID : 6000
  WARN root [2019-09-15 09:43:02,453] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-15 09:43:03,962] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 2244 ----
  INFO root [2019-09-15 09:43:03,962] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-15 09:43:03,962] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-15 09:43:03,963] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-15 09:43:04,378] - ---- cellappmgr is running ----
 DEBUG root [2019-09-15 09:43:04,379] - Components::process(): Request for the process of identity...
  INFO root [2019-09-15 09:43:11,504] - Components::findComponents: find logger(1)...
  INFO root [2019-09-15 09:43:11,528] - Components::findComponents: found logger, addr:192.168.56.1:56468
  INFO root [2019-09-15 09:43:11,528] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 09:43:11,528] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 09:43:11,529] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-15 09:43:11,631] - Components::findComponents: found dbmgr, addr:192.168.56.1:56463
  INFO root [2019-09-15 09:43:11,631] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 09:43:11,631] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 09:43:11,637] - Components::findComponents: found dbmgr, addr:192.168.56.1:56463
  WARN root [2019-09-15 09:43:12,638] - Forwarding logs to logger[192.168.56.1:56468/0/0/0]...
 ERROR root [2019-09-15 09:46:06,516] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=180949704860, lastReceivedTime=180940575904, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-15 09:46:06,520] - The following logs sent to logger failed:
  WARN root [2019-09-15 09:46:06,521] - ==> [2019-09-15 09:46:06 512] TCPPacketReceiver::processPendingEvents(192.168.56.1:56468): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 09:46:06,521] - ==> [2019-09-15 09:46:06 517] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=180949704860, lastReceivedTime=180940575904, inactivityExceptionPeriod=134887325)
  WARN root [2019-09-15 09:46:06,616] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56463): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 09:46:06,618] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=180949935546, lastReceivedTime=180940358671, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-15 09:46:06,635] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-15 09:46:06,638] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-15 09:46:06,638] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-15 09:46:06,638] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-15 09:50:50,536] - 
  INFO root [2019-09-15 09:50:50,537] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-15 09:50:50,538] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-15 09:50:50,539] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-15 09:50:50,539] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-15 09:50:50,573] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56513 )
  INFO root [2019-09-15 09:50:50,573] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56513, SOMAXCONN=5.
  INFO root [2019-09-15 09:50:50,573] - server-configs:
  INFO root [2019-09-15 09:50:50,574] - 	internalTcpAddr : 192.168.56.1:56513
  INFO root [2019-09-15 09:50:50,574] - 	componentID : 6000
  WARN root [2019-09-15 09:50:50,574] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-15 09:50:52,079] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 5136 ----
  INFO root [2019-09-15 09:50:52,079] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-15 09:50:52,080] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-15 09:50:52,080] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-15 09:50:52,473] - ---- cellappmgr is running ----
 DEBUG root [2019-09-15 09:50:52,474] - Components::process(): Request for the process of identity...
  INFO root [2019-09-15 09:50:59,602] - Components::findComponents: find logger(1)...
  INFO root [2019-09-15 09:50:59,636] - Components::findComponents: found logger, addr:192.168.56.1:56511
  INFO root [2019-09-15 09:50:59,636] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 09:50:59,637] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 09:50:59,638] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-15 09:50:59,739] - Components::findComponents: found dbmgr, addr:192.168.56.1:56517
  INFO root [2019-09-15 09:50:59,740] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 09:50:59,740] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 09:50:59,740] - Components::findComponents: found dbmgr, addr:192.168.56.1:56517
  WARN root [2019-09-15 09:51:00,740] - Forwarding logs to logger[192.168.56.1:56511/0/0/0]...
 ERROR root [2019-09-15 09:52:16,578] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=181782346453, lastReceivedTime=181723638794, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-15 09:52:16,583] - The following logs sent to logger failed:
  WARN root [2019-09-15 09:52:16,584] - ==> [2019-09-15 09:52:16 577] TCPPacketReceiver::processPendingEvents(192.168.56.1:56511): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 09:52:16,584] - ==> [2019-09-15 09:52:16 579] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=181782346453, lastReceivedTime=181723638794, inactivityExceptionPeriod=134887325)
  WARN root [2019-09-15 09:52:16,680] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56517): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 09:52:16,688] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=181782592937, lastReceivedTime=181723996464, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-15 09:52:16,704] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-15 09:52:16,709] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-15 09:52:16,710] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-15 09:52:16,711] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-15 09:57:54,890] - 
  INFO root [2019-09-15 09:57:54,891] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-15 09:57:54,892] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-15 09:57:54,892] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-15 09:57:54,892] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-15 09:57:54,908] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56551 )
  INFO root [2019-09-15 09:57:54,908] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56551, SOMAXCONN=5.
  INFO root [2019-09-15 09:57:54,908] - server-configs:
  INFO root [2019-09-15 09:57:54,908] - 	internalTcpAddr : 192.168.56.1:56551
  INFO root [2019-09-15 09:57:54,908] - 	componentID : 6000
  WARN root [2019-09-15 09:57:54,909] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-15 09:57:56,415] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 11632 ----
  INFO root [2019-09-15 09:57:56,416] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-15 09:57:56,417] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-15 09:57:56,417] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-15 09:57:56,802] - ---- cellappmgr is running ----
 DEBUG root [2019-09-15 09:57:56,802] - Components::process(): Request for the process of identity...
  INFO root [2019-09-15 09:58:03,938] - Components::findComponents: find logger(1)...
  INFO root [2019-09-15 09:58:03,957] - Components::findComponents: found logger, addr:192.168.56.1:56557
  INFO root [2019-09-15 09:58:03,957] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 09:58:03,957] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 09:58:03,959] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-15 09:58:04,062] - Components::findComponents: found dbmgr, addr:192.168.56.1:56552
  INFO root [2019-09-15 09:58:04,062] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 09:58:04,062] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 09:58:04,067] - Components::findComponents: found dbmgr, addr:192.168.56.1:56552
  WARN root [2019-09-15 09:58:05,069] - Forwarding logs to logger[192.168.56.1:56557/0/0/0]...
 ERROR root [2019-09-15 10:01:53,380] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=183080150147, lastReceivedTime=183016453493, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-15 10:01:53,381] - The following logs sent to logger failed:
  WARN root [2019-09-15 10:01:53,381] - ==> [2019-09-15 10:01:53 378] TCPPacketReceiver::processPendingEvents(192.168.56.1:56557): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 10:01:53,382] - ==> [2019-09-15 10:01:53 380] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=183080150147, lastReceivedTime=183016453493, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-15 10:01:53,488] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56552): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 10:01:53,493] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=183080406132, lastReceivedTime=183016071151, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-15 10:01:53,504] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-15 10:01:53,510] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-15 10:01:53,510] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-15 10:01:53,511] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-15 10:04:33,759] - 
  INFO root [2019-09-15 10:04:33,791] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-15 10:04:33,792] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-15 10:04:33,792] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-15 10:04:33,793] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-15 10:04:33,823] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56594 )
  INFO root [2019-09-15 10:04:33,823] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56594, SOMAXCONN=5.
  INFO root [2019-09-15 10:04:33,823] - server-configs:
  INFO root [2019-09-15 10:04:33,824] - 	internalTcpAddr : 192.168.56.1:56594
  INFO root [2019-09-15 10:04:33,824] - 	componentID : 6000
  WARN root [2019-09-15 10:04:33,826] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  WARN root [2019-09-15 10:04:34,168] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20201, errorno=10048
  INFO root [2019-09-15 10:04:34,172] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 2448 ----
  INFO root [2019-09-15 10:04:34,173] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-15 10:04:34,173] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-15 10:04:34,173] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-15 10:04:34,565] - ---- cellappmgr is running ----
 DEBUG root [2019-09-15 10:04:34,571] - Components::process(): Request for the process of identity...
  INFO root [2019-09-15 10:04:41,755] - Components::findComponents: find logger(1)...
  INFO root [2019-09-15 10:04:41,782] - Components::findComponents: found logger, addr:192.168.56.1:56591
  INFO root [2019-09-15 10:04:41,782] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 10:04:41,782] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 10:05:46,194] - 
  INFO root [2019-09-15 10:05:46,202] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-15 10:05:46,203] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-15 10:05:46,203] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-15 10:05:46,203] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-15 10:05:46,254] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56609 )
  INFO root [2019-09-15 10:05:46,254] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56609, SOMAXCONN=5.
  INFO root [2019-09-15 10:05:46,254] - server-configs:
  INFO root [2019-09-15 10:05:46,254] - 	internalTcpAddr : 192.168.56.1:56609
  INFO root [2019-09-15 10:05:46,254] - 	componentID : 6000
  WARN root [2019-09-15 10:05:46,256] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-15 10:05:47,761] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 8 ----
  INFO root [2019-09-15 10:05:47,762] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-15 10:05:47,762] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-15 10:05:47,763] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-15 10:05:48,153] - ---- cellappmgr is running ----
 DEBUG root [2019-09-15 10:05:48,153] - Components::process(): Request for the process of identity...
  INFO root [2019-09-15 10:05:55,264] - Components::findComponents: find logger(1)...
  INFO root [2019-09-15 10:05:55,304] - Components::findComponents: found logger, addr:192.168.56.1:56605
  INFO root [2019-09-15 10:05:55,305] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 10:05:55,305] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 10:05:55,306] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-15 10:05:56,408] - Forwarding logs to logger[192.168.56.1:56605/0/0/0]...
 ERROR root [2019-09-15 10:05:58,562] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=183631810369, lastReceivedTime=183624484050, inactivityExceptionPeriod=134887740)
  INFO root [2019-09-15 10:05:58,563] - The following logs sent to logger failed:
  WARN root [2019-09-15 10:05:58,563] - ==> [2019-09-15 10:05:58 560] TCPPacketReceiver::processPendingEvents(192.168.56.1:56605): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 10:05:58,564] - ==> [2019-09-15 10:05:58 563] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=183631810369, lastReceivedTime=183624484050, inactivityExceptionPeriod=134887740)
  INFO root [2019-09-15 10:06:00,206] - 
  INFO root [2019-09-15 10:06:00,206] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-15 10:06:00,207] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-15 10:06:00,208] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-15 10:06:00,208] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-15 10:06:00,224] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56617 )
  INFO root [2019-09-15 10:06:00,224] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56617, SOMAXCONN=5.
  INFO root [2019-09-15 10:06:00,225] - server-configs:
  INFO root [2019-09-15 10:06:00,225] - 	internalTcpAddr : 192.168.56.1:56617
  INFO root [2019-09-15 10:06:00,225] - 	componentID : 6000
  WARN root [2019-09-15 10:06:00,225] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-15 10:06:01,730] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 8968 ----
  INFO root [2019-09-15 10:06:01,731] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-15 10:06:01,731] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-15 10:06:01,731] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-15 10:06:02,121] - ---- cellappmgr is running ----
 DEBUG root [2019-09-15 10:06:02,121] - Components::process(): Request for the process of identity...
  INFO root [2019-09-15 10:06:09,273] - Components::findComponents: find logger(1)...
  INFO root [2019-09-15 10:06:09,290] - Components::findComponents: found logger, addr:192.168.56.1:56620
  INFO root [2019-09-15 10:06:09,290] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 10:06:09,290] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 10:06:09,292] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-15 10:06:10,393] - Forwarding logs to logger[192.168.56.1:56620/0/0/0]...
 ERROR root [2019-09-15 10:06:10,452] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=183658562804, lastReceivedTime=183655952032, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-15 10:06:10,455] - The following logs sent to logger failed:
  WARN root [2019-09-15 10:06:10,455] - ==> [2019-09-15 10:06:10 450] TCPPacketReceiver::processPendingEvents(192.168.56.1:56620): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 10:06:10,455] - ==> [2019-09-15 10:06:10 453] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=183658562804, lastReceivedTime=183655952032, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-15 10:06:28,779] - 
  INFO root [2019-09-15 10:06:28,787] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-15 10:06:28,789] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-15 10:06:28,789] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-15 10:06:28,789] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-15 10:06:28,817] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56637 )
  INFO root [2019-09-15 10:06:28,817] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56637, SOMAXCONN=5.
  INFO root [2019-09-15 10:06:28,817] - server-configs:
  INFO root [2019-09-15 10:06:28,818] - 	internalTcpAddr : 192.168.56.1:56637
  INFO root [2019-09-15 10:06:28,818] - 	componentID : 6000
  WARN root [2019-09-15 10:06:28,820] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  WARN root [2019-09-15 10:06:29,161] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20527, errorno=10048
  INFO root [2019-09-15 10:06:29,165] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 7880 ----
  INFO root [2019-09-15 10:06:29,165] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-15 10:06:29,165] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-15 10:06:29,166] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-15 10:06:29,554] - ---- cellappmgr is running ----
 DEBUG root [2019-09-15 10:06:29,555] - Components::process(): Request for the process of identity...
  INFO root [2019-09-15 10:06:36,699] - Components::findComponents: find logger(1)...
  INFO root [2019-09-15 10:06:36,800] - Components::findComponents: found logger, addr:192.168.56.1:56631
  INFO root [2019-09-15 10:06:36,800] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 10:06:36,800] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 10:06:36,801] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-15 10:06:36,904] - Components::findComponents: found dbmgr, addr:192.168.56.1:56636
  INFO root [2019-09-15 10:06:36,904] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 10:06:36,904] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 10:06:36,904] - Components::findComponents: found dbmgr, addr:192.168.56.1:56636
  WARN root [2019-09-15 10:06:37,905] - Forwarding logs to logger[192.168.56.1:56631/0/0/0]...
 ERROR root [2019-09-15 10:09:18,663] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=184082036850, lastReceivedTime=184037126140, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-15 10:09:18,665] - The following logs sent to logger failed:
  WARN root [2019-09-15 10:09:18,665] - ==> [2019-09-15 10:09:18 661] TCPPacketReceiver::processPendingEvents(192.168.56.1:56631): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 10:09:18,666] - ==> [2019-09-15 10:09:18 663] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=184082036850, lastReceivedTime=184037126140, inactivityExceptionPeriod=134887325)
  WARN root [2019-09-15 10:09:18,775] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56636): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 10:09:18,777] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=184082295337, lastReceivedTime=184037385948, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-15 10:09:18,786] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-15 10:09:18,788] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-15 10:09:18,789] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-15 10:09:18,790] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-15 10:23:15,078] - 
  INFO root [2019-09-15 10:23:15,104] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-15 10:23:15,105] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-15 10:23:15,105] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-15 10:23:15,106] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-15 10:23:15,133] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56679 )
  INFO root [2019-09-15 10:23:15,133] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56679, SOMAXCONN=5.
  INFO root [2019-09-15 10:23:15,134] - server-configs:
  INFO root [2019-09-15 10:23:15,134] - 	internalTcpAddr : 192.168.56.1:56679
  INFO root [2019-09-15 10:23:15,134] - 	componentID : 6000
  WARN root [2019-09-15 10:23:15,147] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-15 10:23:16,651] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 6660 ----
  INFO root [2019-09-15 10:23:16,661] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-15 10:23:16,661] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-15 10:23:16,661] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-15 10:23:17,065] - ---- cellappmgr is running ----
 DEBUG root [2019-09-15 10:23:17,065] - Components::process(): Request for the process of identity...
  INFO root [2019-09-15 10:23:24,202] - Components::findComponents: find logger(1)...
  INFO root [2019-09-15 10:23:24,295] - Components::findComponents: found logger, addr:192.168.56.1:56678
  INFO root [2019-09-15 10:23:24,295] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 10:23:24,295] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 10:23:24,297] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-15 10:23:24,399] - Components::findComponents: found dbmgr, addr:192.168.56.1:56682
  INFO root [2019-09-15 10:23:24,399] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-15 10:23:24,399] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-15 10:23:24,399] - Components::findComponents: found dbmgr, addr:192.168.56.1:56682
  WARN root [2019-09-15 10:23:25,399] - Forwarding logs to logger[192.168.56.1:56678/0/0/0]...
 ERROR root [2019-09-15 10:25:35,376] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=186279644562, lastReceivedTime=186234046501, inactivityExceptionPeriod=134886957)
  INFO root [2019-09-15 10:25:35,379] - The following logs sent to logger failed:
  WARN root [2019-09-15 10:25:35,379] - ==> [2019-09-15 10:25:35 369] TCPPacketReceiver::processPendingEvents(192.168.56.1:56678): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 10:25:35,379] - ==> [2019-09-15 10:25:35 377] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=186279644562, lastReceivedTime=186234046501, inactivityExceptionPeriod=134886957)
  WARN root [2019-09-15 10:25:35,481] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56682): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-15 10:25:35,482] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=186279883309, lastReceivedTime=186234151144, inactivityExceptionPeriod=134886957)
  INFO root [2019-09-15 10:25:35,494] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-15 10:25:35,495] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-15 10:25:35,495] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-15 10:25:35,496] - ThreadPool::destroy(): starting size 2.
